index.html

```diff
@@ -70,10 +70,10 @@ <h2 id="main-heading">My Recent WORKS</h2>
 
     <div class="card-holder">
 
-        <div class="card-holder-item">
+        <!-- <div class="card-holder-item">
```

Comment:
- [ ] [OPTIONAL] Kindly ensure you don't have commented code in your files, its not a good practice